api/namespace/application/models/application.py

```diff
@@ -0,0 +1,117 @@
+from api.namespace.application.application_ns import application_ns
+from api.namespace.application.models.question import question
+from api.namespace.application.models.question import question_status
+from api.namespace.application.models.metadata import metadata
+
+from flask_restx import fields
+
+application_inbound = application_ns.model(
+    "application_inbound",
+    {
+        "name": fields.String(
+            required=True,
+            description="The name of the fund",
+            example="Funding Service Design"
+        ),
+        "questions": fields.List(
+            fields.Nested(
+                question,
+                required=True,
+                description=(
+                    "The payload of application questions and"
+                    " answers."
+                ),
+            ),
+        ),
+        "metadata": fields.Nested(
+            metadata,
+            description="Application metadata"
+        ),
+    },
+)
+
+application_full = application_ns.model(
+    "application_full",
+    {
+        "id": fields.String(
+            description="The id of the application.",
+        ),
+        "status": fields.String(
+            description="The status of the application",
+            example="NOT_STARTED"
+        ),
+        "fund_id": fields.String(
+            description="The id of the fund",
+            example="funding-service-design"
+        ),
+        "fund_name": fields.String(
```

Comment:
Do we need this? we have the fund_id - if the fund name changes we have to update it here as well - the fund-store should be our source of truth for all information about the fund

Reply:
Ok, appreciate that single source of truth, however when we're pulling a list of applications from multiple funds (as the assessment frontend does), if we wanted fund_name for each application we would have to make a separate api call for each row (or cache fund_store data on assessment frontend).

Reply:
Is there a preferred way for collating info like this for related objects when returning a list?

Reply:
Seperate API calls are expected!